README.md

```diff
@@ -46,21 +48,22 @@ Then follow the instructions in [INSTALL](INSTALL.md).
 
 ## Configuration
 
-For necessary server-side configurations, follow instructions in
-[INSTALL - Configuration](INSTALL.md#configuration).
+If not using the vagrant box, follow instructions in
+[INSTALL - Stand alone installation](INSTALL.md#stand-alone-installation)
+for necessary server-side configurations.
 
 ## Usage
 
-Generally you will have four tabs (or windows) open in your terminal when using this project.
-These will be used for:
+When running this project you will generally have four tabs
+(or windows) open in your terminal, which will be used for:
  1. **Git operations**.
     Perform Git operations and general development in the repository.
  2. **Elasticsearch**.
     Run an Elasticsearch (ES) instance.
-    Running `cfgov/manage.py sheer_index` will load indexes into ES.
- 3. **Django server**. Start and run the web server.
+ 3. **Django server**. Start and stop the web server.
  4. **Gulp watch**.
-    Run the Gulp watch task for watching for changes to content.
+    Run the Gulp watch task to automatically re-run the gulp
+    asset compilation tasks when their source files are changed.
 
```

Comment:
it might be helpful to provide the specific commands here for steps 2-4 (e.g. `python cfgov/manage.py runserver`) etc.